azure-ts-api-management/index.ts

```diff
@@ -1,8 +1,13 @@
 import * as pulumi from "@pulumi/pulumi";
 import * as azure from "@pulumi/azure";
 
+// use first 10 characters of the stackname as prefix for resource names
+const prefix = pulumi.getStack().substring(0, 9);
+
 // Create an Azure Resource Group
-const resourceGroup = new azure.core.ResourceGroup("resourceGroup");
+const resourceGroup = new azure.core.ResourceGroup(`${prefix}-rg`, {
```

Comment:
Why is the prefix needed?

Reply:
It's not - this was something I will remove on next PR